gutenberg.php

```diff
@@ -122,7 +122,7 @@ function gutenberg_pre_init() {
 	// Strip '-src' from the version string. Messes up version_compare().
 	$version = str_replace( '-src', '', $wp_version );
 
-	if ( version_compare( $version, '5.0.0', '<' ) ) {
+	if ( version_compare( $version, '5.2.0', '<' ) ) {
```

Comment:
The change to increase the minimal version of WordPress and PHP could be extracted as its own PR and landed soon.

Reply:
@gziolo We already have #15809 which increases the minimum version to 5.2.

Reply:
Let's land it then 💯

Reply:
> The change to increase the minimal version of WordPress and PHP could be extracted as its own PR and landed soon.

@gziolo @Soean Thanks, I'll make sure this is removed in a rebase.

For context, it was only needed due to lack of support of non-numeric values for [`array_reduce`](https://www.php.net/manual/en/function.array-reduce.php) third parameter. Not a big issue to refactor if it came to it, but I assumed (and supported by #15809!) that it would be fine to bump the minimum requirement.